docs/api/environment-variables.md

```diff
@@ -19,6 +19,35 @@ Windows console example:
 > electron
 ```
 
+## Production Variables
+
+The following environment variables are intended primarily for use at runtime
+in packaged Electron applications.
+
+### `GOOGLE_API_KEY`
+
+Electron includes a hardcoded API key for making requests to Google's geocoding
+webservice. Because this API key is included in every version of Electron, it
+often exceeds its usage quota. To work around this, you can supply your own
+Google API key in the environment. Place the following code in your main process
+file, before opening any browser windows that will make geocoding requests:
+
+```javascript
+process.env.GOOGLE_API_KEY='YOUR_KEY_HERE'
```

Comment:
Spaces needed around the `=`

Reply:
> 
> 
> fixed. thanks.

hey i am creating an electron app which require to fetch users geocoord. but normally by js its not working can u help? how to get geolocation in electron